src/decide.js

```diff
@@ -505,6 +508,40 @@ const fetchPulls = async ({ baseUrl, owner, name, pullsMode, query }) => {
   return pulls
 }
 
+// fetchFullPulls gets full data for each pull request provided
+const fetchFullPulls = async ({ baseUrl, owner, name, pulls }) => {
+  const fullPulls = []
+
+  for (let pull of pulls) {
+    try {
+      const url = `${baseUrl}/repos/${owner}/${name}/pulls/${pull.number}`
+
+      const { res, data: fullPull } = await githubFetch(url)
+
+      log({ pullNum: pull.html_url, res, fullPull })
+
+      // carry over labels from search results, since full pull doesn't have
+      fullPull.labels = pull.labels
+
+      fullPulls.push(fullPull)
+    } catch (err) {
+      trace(err)
+    }
+  }
+
+  return fullPulls
+}
+
+const isStateUnknown = pull => pull.mergeable_state === 'unknown'
+
+const allMergeableStateUnknown = pulls => {
+  if (pulls === undefined || pulls.length === 0) {
```

Comment:
I would just do `!pulls || pulls.length === 0` here, or even use `lodash/isEmpty` like `isEmpty(pulls)` here, but this is fine. JavaScript types are a cluster.